src/Sarif/FileRegionsCache.cs

```diff
@@ -65,8 +73,13 @@ public virtual Region PopulateTextRegionProperties(Region inputRegion, Uri uri,
                 return inputRegion;
             }
 
-            NewLineIndex newLineIndex = GetNewLineIndex(uri, out string fileText);
-            return PopulateTextRegionProperties(newLineIndex, inputRegion, fileText, populateSnippet);
+            NewLineIndex newLineIndex = GetNewLineIndex(uri, fileText);
+
+            return PopulateTextRegionProperties(
+                newLineIndex,
+                inputRegion,
+                newLineIndex?.Text,
```

Comment:
>newLineIndex?.Text [](start = 16, length = 18)

this could generate a null reference. Looked at the method and the other calls, we are not protecting it against null. Should we default empty?